### PR TITLE
Print output of predictions in simple estimator example for DistributionStrategy

### PR DESCRIPTION
**Pull request by jppgks**
Resolved the TODO (cc @anj-s) that asked to print meaningful results from the Estimator's predictions.
Collects all elements yielded by the generator in a list and prints that list instead of the generator object.
Results in the following output: `Prediction results: [{'logits': array([1.0162734], dtype=float32)}, ..., {'logits': array([1.0162734], dtype=float32)}]`.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

File: tensorflow/contrib/distribute/python/examples/simple_estimator_example.py [hunk 1]
```diff
@@ -78,9 +78,8 @@ def predict_input_fn():
     predict_features = tf.data.Dataset.from_tensors([[1.]]).repeat(10)
     return predict_features
 
-  predictions = estimator.predict(input_fn=predict_input_fn)
-  # TODO(anjalsridhar): This returns a generator object, figure out how to get
-  # meaningful results here.
+  prediction_iterable = estimator.predict(input_fn=predict_input_fn)
+  predictions = [prediction_iterable.next() for _ in range(10)]
```
Comment (anj-s) [inline tensorflow/contrib/distribute/python/examples/simple_estimator_example.py]: Thanks for adding this! Can we add a comment that predictions is a list of output dicts with key as 'logits' and value as an array of model outputs?

**Review by anj-s — APPROVED** [review]
Looks good. Thanks!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 33 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 48 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 63 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 78 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 93 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
